livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -69,7 +71,6 @@ public void open() {
     } catch (LivyException e) {
       String msg = "Fail to create session, please check livy interpreter log and " +
           "livy server log";
-      LOGGER.error(msg);
```

Comment:
It is not necessary, because all the exceptions will be caught by Job.java

```
JobProgressPoller progressUpdator = null;
    try {
      progressUpdator = new JobProgressPoller(this, progressUpdateIntervalMs);
      progressUpdator.start();
      dateStarted = new Date();
      setResult(jobRun());
      this.exception = null;
      errorMessage = null;
      dateFinished = new Date();
      progressUpdator.terminate();
    } catch (NullPointerException e) {
      LOGGER.error("Job failed", e);
      progressUpdator.terminate();
      this.exception = e;
      setResult(e.getMessage());
      errorMessage = getStack(e);
      dateFinished = new Date();
    } catch (Throwable e) {
      LOGGER.error("Job failed", e);
      progressUpdator.terminate();
      this.exception = e;
      setResult(e.getMessage());
      errorMessage = getStack(e);
      dateFinished = new Date();
    } finally {
      //aborted = false;
    }
```